tests/foreman/cli/test_host.py

```diff
@@ -329,6 +330,115 @@ def test_positive_create_with_org_name(self):
         })
         self.assertEqual(new_host['organization'], self.new_org['name'])
 
+    @skip_if_bug_open('bugzilla', 1483252)
+    @tier1
+    def test_positive_create_with_content_source(self):
+        """Create a host with content source specified
+
+        :id: 6068bd4d-18d8-47a2-99f4-3e0ee9208104
+
+        :BZ: 1260697, 1483252
+
+        :expectedresults: A host is created with expected content source
+            assigned
+
+        :CaseImportance: High
+        """
+        content_source = Proxy.list({
+            'search': 'url = https://{0}:9090'.format(settings.server.hostname)
+        })[0]
+        host = make_fake_host({
+            'content-source-id': content_source['id'],
+            'content-view-id': self.DEFAULT_CV['id'],
+            'lifecycle-environment-id': self.LIBRARY['id'],
+            'organization': self.new_org['name'],
+        })
+        self.assertEqual(host['content-source'], content_source['name'])
+
+    @tier1
+    def test_negative_create_with_content_source(self):
+        """Attempt to create a host with invalid content source specified
+
+        :id: d92d6aff-4ad3-467c-88a8-5a5e56614f58
+
+        :BZ: 126069
```

Comment:
How this test is related to BZ 126069

Reply:
It's typo, should be 1260697, thanks for pointing.